actions/bump-version/README.md

```diff
@@ -20,6 +20,7 @@ current_version = 0.0.1
 | ----------------- | :------: | :-----------: | ----------------------------------------------------- |
 | release-type      |    ✅    |       -       | The scope of the release (major, minor or patch)      |
 | working-directory |    ❌    |       .       | The directory containing the `.bumpversion.cfg` file. |
+| new-version       |    ❌    |       .       | New version to use for the bump                       |
```

Comment:
is there any way of making `release-type` and `new-version` mutually exclusive? afaict only one of the options should be used, right?

Reply:
not really the `bump2version` cmd expect a `release-set` even with  the flag `--new-version` set

Reply:
ah ok, that's a bit weird, but guess there's nothing we can do about it